Sources/ArcGISToolkit/FloorFilter/SiteFacilityPromptViewController.swift

```diff
@@ -0,0 +1,277 @@
+//
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+import Foundation
+
+internal class SiteFacilityPromptViewController: UIViewController {
+    
+    var delegate: FloorFilterViewControllerDelegate?
+    var viewModel = FloorFilterViewModel()
+    
+    // UI Elements and constraints
+    @IBOutlet weak var siteFacilitySearchBar: UISearchBar!
+    @IBOutlet weak var backBtn: UIImageView!
+    @IBOutlet weak var closeBtn: UIImageView!
```

Comment:
I made these imageViews because they are an image with the back icon and close icon.

Reply:
Out of context - if it's possible, consider using [SF Symbols](https://developer.apple.com/sf-symbols/) for button icons. 🙂

Reply:
I agree with @mhdostal; these should be buttons so that they look and behave as users would expect. Use `UIButton.setImage(_:for:)` to set the image for the `normal` state.